gcloud-java-pubsub/src/main/java/com/google/gcloud/pubsub/spi/PublisherApi.java

```diff
@@ -0,0 +1,498 @@
+/*
```

Comment:
Any reason for using a difference copyright?
Would that be an issue with our Apache 2.0 license?